cgi-bin/DW/EmailPost/Entry.pm

```diff
@@ -513,7 +512,7 @@ sub _cleanup_mobile_carriers {
         return $self->err( "Unable to find XML content in PictureMail message." )
             unless $xml_string;
 
-        HTML::Entities::decode_entities( $xml_string );
+        LJ::dhtml( $xml_string );
```

Comment:
Can you add a comment here to indicate that $xml_string is being modified by the function call?  It's documented in the POD for HTML::Entities, but it's unusual for a function to behave this way in Perl.